Realm/Realm/Schema/ObjectSchema.cs

```diff
@@ -145,31 +174,64 @@ public class Builder : SchemaBuilderBase<Property>
             /// Gets or sets a value indicating whether this <see cref="Builder"/> describes an embedded object.
             /// </summary>
             /// <value><c>true</c> if the schema pertains to an <see cref="EmbeddedObject"/> instance; <c>false</c> otherwise.</value>
-            public bool IsEmbedded { get; set; }
+            [Obsolete("Check against RealmSchemaType instead.")]
+            public bool IsEmbedded
+            {
+                get
+                {
+                    return RealmSchemaType == ObjectType.EmbeddedObject;
+                }
+
+                set
+                {
+                    // being an obsolete method, it's assumed that it's old code that doesn't use AsymmetricObjects
+                    _ = value ? RealmSchemaType = ObjectType.EmbeddedObject : RealmSchemaType = ObjectType.RealmObject;
```

Comment:
That is super awkward - let's simplify to

```suggestion
                    RealmSchemaType = value ? ObjectType.EmbeddedObject : ObjectType.RealmObject;
```

Reply:
Emh.... 🤦🏻